src/Client.php

```diff
@@ -78,6 +83,8 @@ class Client
      *
      * Supported driver-specific options:
      *
+     *  * builderEncoder (MongoDB\Builder\Encoder): Encoder for query builder
```

Comment:
Suggestion to call out there's a default value being used:
```suggestion
     *  * builderEncoder (MongoDB\Builder\Encoder): Encoder for query and aggregation builders. If not given, the default encoder will be used.
```

If you accept this, it would make sense to apply this to the Database and Collection classes as well.